### PR TITLE
AC Test Fix - LRAC-14798 | master

### PR DESCRIPTION
**Pull request by ViniLopes20**
https://liferay.atlassian.net/browse/LRAC-14798

### CODE REVIEW

**Comment by ViniLopes20** [comment]
ci:test:sf

**Review by nkkwon — APPROVED** [review]
LGTM! :+1:

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/CustomEventsSavingAnalysis.testcase [hunk 1]
```diff
@@ -711,18 +711,7 @@ definition {
...
-			var customEventName = ACCustomEvents.generateCustomEventName();
-
-			ACCustomEvents.createCustomEvent(
-				customEventName = ${customEventName},
-				timeToSendCustomEvent = 1);
-		}
-
-		task ("Close sessions") {
-			ACUtils.closeAllSessionsAndWait();
-		}
```
Comment (manuferreira) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/CustomEventsSavingAnalysis.testcase:725]: Hey Vini, can you explain to me why you are deleting these lines of code (same thing with the other lines)? Isn't this part of the test?

Comment (ViniLopes20) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/CustomEventsSavingAnalysis.testcase:725]: Hey @manuferreira, The purpose of this test is to check if the pagination of the event analysis list is working or not. So it is not necessary to create a custom event to check this, so I removed this part and put the test to create the event analysis using the default events.

I also removed an assert that it made to check if the event analysis had data before creating it. For this test case, it is not important if we have data in the event analysis or not.

I believe that this way we can make the test more simple and targeted to the objective and avoid problems that could happen by generating the custom event (sometimes the emulator dies when we generate the custom event)

What do you think?

Comment (manuferreira) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/CustomEventsSavingAnalysis.testcase:725]: sounds good to me!

**Review by manuferreira — APPROVED** [review]
LGTM. Thanks for working on this! :muscle:

**Review by joshitagaki — APPROVED** [review]
LGTM! Thanks Vini!

**Comment by joshitagaki** [comment]
ci:forward

**Pull request closed without merging**